Models/Courses.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.declarative import declarative_base
+from  sqlalchemy import Column, Integer, String
+
+
+Base = declarative_base()
+
+
+class course(Base):
+    __tablename__ = 'courses'
+    department_ids = Column(String)
```

Comment:
I don't think the database supports me to write arrays. Later on, I will wrap this and present it as a list.